docs/articles/expensify-classic/travel/Edit-or-cancel-travel-arrangements.md

```diff
@@ -3,9 +3,55 @@ title: Edit or cancel travel arrangements
 description: Modify travel arrangements booked with Expensify Travel
 ---
 <div id="expensify-classic" markdown="1">
+It's easy to change/cancel a flight booked with Expensify Travel. 
 
-Click **Get Support** on your emailed travel itinerary for real-time help with the booking. Any modifications, exchanges, or voidings made to a trip via support will incur a $25 booking change fee. 
+To modify or cancel a travel arrangement,
+<br>
+1. Navigate to Expensify Travel on an internet browser by clicking **Travel** and then **Book or Manage Travel**
+   ![image](https://github.com/Expensify/App/assets/3697042/c2d0959e-3428-489b-80ee-bfc3477fe96e)
 
+<br>
+<br>
```

Comment:
can you remove all of these extra line breaks? there's a ton of extra space on the page as a result

Reply:
The spacing looks bad without it